apps/site/assets/ts/schedule/components/schedule-finder/UpcomingDepartures.tsx

```diff
@@ -25,19 +28,23 @@ interface Props {
   input: UserInput;
 }
 
+interface AccordionProps {
+  state: State;
+  journey: EnhancedJourney;
+  contentComponent: () => ReactElement<HTMLElement>;
+}
+
+type fetchAction =
```

Comment:
Could this please be capitalized to distinguish it as a type?

```suggestion
type FetchAction =
```